src/ekobox/utils.py

```diff
@@ -35,9 +35,10 @@ def ekos_product(eko_ini, eko_fin, in_place=True, force=False):
     final_dict = {}
     for Q2 in ope2_dict.keys():
         final_op_dict[Q2] = np.einsum("ajbk,bkcl -> ajcl", ope1, ope2_dict[Q2])
-        final_op_error_dict[Q2] = np.einsum(
-            "ajbk,bkcl -> ajcl", ope1_error, ope2_error_dict[Q2]
-        )
+        final_op_error_dict[Q2] = np.add(np.einsum(
```

Comment:
If you wish, `np.add` is equivalent to `+` for `np.ndarray`s

Reply:
Yes I know (it is in the docs) but I had some problems with that. That's why I prefer doing like this. Anyway I can change if you prefer.

Reply:
I did it, since I guessed the problems were only syntactical. In any case, can you please test it that is still fine for you?